src/alerts/version-polling.test.ts

```diff
@@ -63,40 +44,17 @@ describe('checkVersion', () => {
       expect(getBadVersions).toHaveBeenCalled();
     });
 
-    it('reloads the page if current version is bad', async () => {
+    it('sets updated required flag if current version is bad', async () => {
       // Arrange
       asMockedFn(getBadVersions).mockResolvedValue(['abcd123']);
 
       // Act
       await checkVersion();
 
       // Assert
-      expect(window.location.reload).toHaveBeenCalled();
-    });
-
-    it('does not reload the page if current version is good', async () => {
-      // Arrange
-      asMockedFn(getBadVersions).mockResolvedValue([]);
-
-      // Act
-      await checkVersion();
-
-      // Assert
-      expect(window.location.reload).not.toHaveBeenCalled();
+      expect(versionStore.get()).toMatchObject({ isUpdateRequired: true });
```

Comment:
Also a test for when it returns `isUpdatedRequired` as false because the current version is not a "bad version"?